sdk/src/connections/offline/offline_depth_sensor.cpp

```diff
@@ -1,7 +1,7 @@
 #include "offline_depth_sensor.h"
 #include "tofi/tofi_config.h"
 #include <cstring>
-#ifdef UNIX
+#ifdef __linux__
```

Comment:
The commit addressing these changes says "Solved offline problem on linux".
So, what was the problem? Can you describe it?

Reply:
"UNIX" was the wrong identifier for linux. I've changed it to "__linux__" so the code will run correctly when running on linux.